lxd/storage/backend_lxd.go

```diff
@@ -7495,6 +7495,68 @@ func (b *lxdBackend) CheckInstanceBackupFileSnapshots(backupConf *backupConfig.C
 	return backupConf.Snapshots, nil
 }
 
+// resolveAndSkipUnknownVolumes tries to resolve volumes returned from a driver's ListVolumes.
+// In addition it skips unknown custom volumes which cannot be recovered by name in case its transformed.
+// In case a driver is using volumes with transformed names, we have to identify which instance in LXD
+// belongs to the respective volumes.
+// This is necessary to not disturb potentially running instances in case we are trying to re-mount an instance's volume
+// which is already mounted to the system by its actual name which we don't yet know because we haven't looked into
+// the instances backup config file.
+// Therefore try to lookup the instance's root volume using the volume's UUID.
+// An instance's root volume always has the same name as its parent instance.
+func (b *lxdBackend) resolveAndSkipUnknownVolumes(poolVols []drivers.Volume) ([]drivers.Volume, error) {
+	resolvedUnknownVols := make([]drivers.Volume, 0, len(poolVols))
+
+	for _, vol := range poolVols {
+		volUUID := vol.Config()["volatile.uuid"]
+
+		if vol.Name() == "" {
+			return nil, fmt.Errorf("Cannot detect instance with UUID %q on pool %q using an empty volume name", volUUID, b.name)
```

Comment:
this error doesnt correlate with the check, its checking volume name and complaining about UUID, furthermore this check isn't restricted to only pools with `UUIDVolumeNames`, so doesn't make sense in a general context.

Reply:
uuid is fine by me